components/scream/src/share/atm_process/atmosphere_process_dag.hpp

```diff
@@ -38,18 +39,25 @@ class AtmProcDAG {
   // If already stored, simply return its position
   int add_fid (const FieldIdentifier& fid);
 
+  int get_fid_id (const FieldIdentifier& fid) const;
```

Comment:
Is this "field identifier ID" supposed to be an ID, or an index (of the identifier within a list, say)? If the latter, maybe we could call this `get_fid_index`.

Reply:
Good point. I'll change this.